src/main/java/org/liquidengine/legui/component/Component.java

```diff
@@ -95,6 +98,19 @@
      */
     private boolean tabFocusable = true;
 
+    ////////////////////////////////
+    //// COMPONENT LAYER DATA
+    ////////////////////////////////
+
+    /**
+     * Layout. Used to layout
+     */
+    private Layout layout = new BorderLayout();
```

Comment:
Why is BorderLayout the default? why not null as default?

Reply:
I'll change that.

Reply:
I thought that not experienced users should face with something complicated :D